models/message.js

```diff
@@ -61,4 +61,10 @@ messageSchema.post('save', async function (doc) {
   await Chat.updateOne({ _id: chat }, { latestMessage: _id })
 })
 
+messageSchema.post('remove', async function (doc) {
+  const { chat } = doc
+
+  await Chat.updateOne({ _id: chat }, { latestMessage: null })

```

Comment:
I guess, that leaves latestMessage as null is bad. In my opinion, you better leave a previous message in this chat. 
As an example, we can display a list of chats on the left pane, which includes the user's name, image, and their most recent message.